.github/workflows/tests.yml

```diff
@@ -109,16 +109,9 @@ jobs:
           echo "debian_release=$DEBIAN_RELEASE" >> $GITHUB_OUTPUT
         shell: bash
 
-      - name: Bundler cache
-        uses: actions/cache@v4
-        if: github.repository != 'discourse/discourse'
-        with:
-          path: vendor/bundle
-          key: ${{ runner.os }}-${{ steps.container-envs.outputs.ruby_version }}-${{ steps.container-envs.outputs.debian_release }}-gem-${{ hashFiles('**/Gemfile.lock') }}-cachev2
```

Comment:
No longer need to rely on Github actions cache cause the gems are indirectly "cached" in the `discourse/discourse_test:release` image.